queries.sql

```diff
@@ -1,7 +1,18 @@
 -- Part 1: Test it with SQL
+id INT
+employer VARCHAR
+name VARCHAR
+skills VARCHAR
 
 -- Part 2: Test it with SQL
+SELECT name
+FROM employer
+where location = "St. Louis";
 
 -- Part 3: Test it with SQL
+DROP TABLE job;
 
--- Part 4: Test it with SQL
\ No newline at end of file
+-- Part 4: Test it with SQL
+SELECT name, description
+FROM skill
+INNER join job_skills on skill.id = job_skills.skills_id
```

Comment:
```suggestion
SELECT *
FROM skill
INNER join job_skills on skill.id = job_skills.skills_id
```
Your query is pretty close to being correct. For the test associated with this query, it's expecting to return the entire table from skill where it is filtering out null values and only finding ones attached to jobs. You'll need to also add a command to order the table in ascending order.